packages/embark-accounts-manager/src/index.ts

```diff
@@ -0,0 +1,166 @@
+import async from "async";
+import {Embark, Events, Logger} /* supplied by @types/embark in packages/embark-typings */ from "embark";
+import {__} from "embark-i18n";
+import {AccountParser, dappPath} from "embark-utils";
+import Web3 from "web3";
+
+import fundAccount from "./fundAccount";
+
+export default class AccountsManager {
+  private readonly logger: Logger;
+  private readonly events: Events;
+  private accounts: any[];
+  private nodeAccounts: string[];
+  private _web3: any;
```

Comment:
Does `web3` not have a typing (ie `@types/web3`)? Also, could we please default it to null (avoids any falsy checking issues in the `web3` property if `_web3` is typed to `Web3`).
```javascript=
import Web3 from "@types/web3";
export default class AccountsManager {
  // ...
  private _web3: Web3 = null;
  // ...
}
```

Reply:
Ok, thanks for the tips you two. Will do.